Src/CSharpier.Tests/TestFiles/BaseTest.cs

```diff
@@ -31,8 +31,11 @@ protected void RunTest(string folderName, string fileName, bool useTabs = false)
                 folderName,
                 fileName + ".cst"
             );
-            var fileReaderResult =
-                FileReader.ReadFile(filePath, new FileSystem(), CancellationToken.None).Result;
+            var fileReaderResult = FileReader.ReadFile(
```

Comment:
This should be just temporary. The idea is that with a conditional group we can say
```c#
//format this way if it fits (this would use a Line after = to keep it on a single line when possible)
var someObject =
    CallSomeMethod(withSomeParameters, likeSo);
// and if that doesn't fit, then try
var someObject = CallSomeMethod(
    withSomeParameters,
    likeSo
);
```
Without conditional groups when it gets too long it would end up as
```c#
var someObject =
    CallSomeMethod(
        withSomeParameters,
        likeSo
    );

Reply:
to be perfectly honest, I don't think the 3rd case looks too bad. But I also don't feel strongly enough to push _for_ it either.

Instead of a conditional group, can you use a `BreakParent()` combined with an `IfBreak()` to achieve that?

Reply:
> Instead of a conditional group, can you use a BreakParent() combined with an IfBreak() to achieve the second case?
It might be possible. In my head it seems like it could work.
I know prettier uses conditional groups for this type of formatting, so I've been assuming we are going to need them as well. It is definitely worth trying out alternatives before using conditional groups. I believe they add a lot of extra computation to the printing process when they are used.